src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,99 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final int CINEMA_HALL_CAPACITY = 50;
+    private static final int FATS_AND_FURIOUS_MOVIE = 0;
```

Comment:
There is a typo in the constant name `FATS_AND_FURIOUS_MOVIE`. It should be `FAST_AND_FURIOUS_MOVIE` to accurately reflect the movie title 'Fast and Furious'.